### ISSUE TITLE
[meta] reddit bugs

### ISSUE THREAD

**Issue by paulrouget** [servo/servo#9917]
We want to properly support reddit.com. It needs to behave and be rendered properly (frontpage, comments, login).

See https://github.com/servo/servo/wiki/WebCompatBug

---

- [x] #10012
- [x] https://github.com/servo/servo/issues/10027
- [x] #12094

**Comment by pygy** [comment]
You may want to limit the scope of this bug to the default subreddits, because comments sections can have custom (moderator-mrovided) CSS.

If you don't, I know of at least one sneaky bastard who might be tempted to turn his own broken page into a subreddit to bump the priority of his issues ;-)

**Comment by paulrouget** [comment]
Depends on: https://github.com/servo/servo/issues/10012

**Comment by pcwalton** [comment]
Is this fixed?

**Comment by bheesham** [comment]
New issue related to Reddit: https://github.com/servo/servo/issues/12094

**Comment by paulrouget** [comment]
This was a tracking bug for nightly builds. Closing now.
